package.json

```diff
@@ -9,6 +9,7 @@
     "@reach/accordion": "^0.13.2",
     "@reach/checkbox": "^0.13.2",
     "@reach/combobox": "^0.13.2",
+    "@reach/dialog": "^0.15.0",
```

Comment:
Is this for a separate PR?

Reply:
yeah i added it to my other branch but it added it here, should I delete it?

Reply:
Yes.